web/themes/custom/itkdev/itkdev_project_theme/templates/components/project-status.html.twig

```diff
@@ -12,40 +12,34 @@
 #}
 
 {% set evaluation = project_status.track_evaluation|default([0]) %}
-{% set project_status_max = max(evaluation) %}
-{% set project_status_min = min(evaluation) %}
 
 <div class="flex flex-col gap-1 w-20">
     <div class="w-full h-2
-        {% if project_status_min == 0 %}
-            bg-zinc-400
-        {% elseif project_status_max >= 1 %}
-            {% if project_status_max == 1 %}
-                bg-green-600
-            {% endif %}
-            {% if project_status_max == 2 %}
-                bg-yellow-600
-            {% endif %}
-            {% if project_status_max == 3 %}
-                bg-red-600
-            {% endif %}
+        {% if 'none' in evaluation %}
+          bg-zinc-400
+        {% elseif 'refused' in evaluation %}
+          bg-red-600
+        {% elseif 'undecided' in evaluation %}
+          bg-yellow-600
+        {% elseif 'approved' in evaluation %}
+          bg-green-600
```

Comment:
Consider using the same tricks as you use in `project-track-edit-form.html.twig` here, e.g.

``` twig
{{ {
none: 'bg-zinc-400',
refused: 'bg-red-600',
undecided: 'bg-yellow-600'
approved: 'bg-green-600',
}[evaluation] ?? '' }}
```

Setting a `className` variable, say, may make the code easier to read and understand.